next/components/contact-list.tsx

```diff
@@ -0,0 +1,75 @@
+import NextImage from "next/image";
+import { useTranslation } from "next-i18next";
+
+interface Contact {
+  image: string;
+  name: string;
+  title: string;
+  phoneNumber: string;
+  email: string;
+  id: number;
+}
+
+const contacts: Contact[] = [
+  {
+    image: "/john_dean.jpg",
+    name: "John Dean",
+    title: "CFO",
+    phoneNumber: "+358445123456",
+    email: "john.dean@mail.com",
```

Comment:
Looks like `mail.com` is a real thing, let's instead use example.com which is reserved for examples?

Reply:
done.